styles/shared/timeline-post.scss

```diff
@@ -27,5 +27,14 @@
         margin: 6px 0 12px;
       }
     }
+    
+    .p-post-actions {
+      text-align: right;
+    }
+
+    .p-post-cancel {
+      margin-right: 10px;
+      margin-left: 5px;
+    }
```

Comment:
`.p-*` classes (from the old Ember client) existed solely for the purpose of automated testing. We should never use them for styling.